src/app/public/modules/radio/radio-group.component.ts

```diff
@@ -97,6 +125,14 @@ export class SkyRadioGroupComponent implements AfterContentInit, AfterViewInit,
     return this._value;
   }
 
+/**
+ * Specifies an index for all the radio buttons in the group. If the index is not defined,
+ * the indices for individual radio buttons are set to their positions on load.
+ * This property supports
```

Comment:
This last sentence is long and unwieldy. I'm not sure of the best way to fix it, but I wonder if it should be two sentences:

``` This property supports accessibility by placing focus on the currently selected radio button. If no radio button is selected, it places focus on the first or last button depending on how users navigate to the radio button group.```

What do you think?

Reply:
I like that change. Done.